tar.go

```diff
@@ -54,11 +62,17 @@ func tar(ctx context.Context, enc FormatEncoder, path string, info os.FileInfo)
 	)
 
 	switch sys := info.Sys().(type) {
+	case *arctar.Header:
```

Comment:
Not quite sure where this comes from. Could you explain a bit more of when that code path would be used?

Reply:
It was an initial attempt of another feature's implementation: allow desync to use a .tar file instead of filesystem tree as a "tar" source (issue #84).